components/teleport/Teleport.vue

```diff
@@ -177,11 +211,32 @@ const toTabs = [
     value: Chain.STATEMINE,
     disabled: computed(() => isDisabled(Chain.STATEMINE)),
   },
+  {
+    label: Chain.POLKADOT,
+    value: Chain.POLKADOT,
+    disabled: computed(() => isDisabled(Chain.POLKADOT)),
+  },
+  {
+    label: Chain.STATEMINT,
+    value: Chain.STATEMINT,
+    disabled: computed(() => isDisabled(Chain.STATEMINT)),
+  },
 ]
 
-const ksmTokenDecimals = computed(() => assets(5).decimals)
+const currentTokenDecimals = computed(() => {
+  switch (fromChain.value) {
+    case Chain.KUSAMA:
+    case Chain.BASILISK:
+    case Chain.STATEMINE:
+      return assets(5).decimals
```

Comment:
`const { tokenId } = usePrefix()` instead of `5`?

Reply:
The tokenId from `usePrefix ` is based on the prefix on the url. It's not the same chain as what the user selected on teleport page.